src/style.css

```diff
@@ -1,3 +1,59 @@
-body {
+html {
+  font-family: "Roboto", sans-serif;
+  --secondaryColor: #00acdc;
+}
+
+.header {
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+}
+
+img {
```

Comment:
use selector by class, you have `img__link` for it [here](https://github.com/mate-academy/layout_moyo-header/pull/3754/files#diff-ce434ef5cc161194b450135a3586eb89357fc270e8aaa8e336db5a04e8d52fbeR24)

Reply:
only this left